File: Java/1011.java
import java.util.Scanner;
import java.lang.Math;

public class Main {

  public static void main(String[] args)  {
    Scanner scanner = new Scanner(System.in);
    
    double pi = 3.14159;
    int raio = scanner.nextInt();
    
    double volume = (4/3.0) * (pi * Math.pow(raio,3)); 

    String volumeFormatado = String.format("%.3f", volume);
    
    System.out.println("VOLUME = " + volumeFormatado);
  }
}

File: Javascript/1011.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var raio = parseInt(lines.shift());
var pi = 3.14159
var volume = (4/3) * pi * Math.pow(raio, 3)

console.log("VOLUME = " + volume.toFixed(3))